test/integration/collector-remote-method.test.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
```

Comment:
so this test isn't getting run in CI because of `'test/integration/**/*.tap.js'`

Reply:
Good catch. Should be fixed.